ctapipe/instrument/subarray.py

```diff
@@ -81,8 +81,8 @@ def __init__(
         ----------
         name : str
             name of this subarray
-        tel_positions : Dict[int, np.ndarray]
-            dict of x,y,z telescope positions on the ground by tel_id. These are
+        tel_positions : Dict[int, Union[np.ndarray, EarthLocation]]
+            dict of x,y,z telescope positions on the ground by tel_id or EarthLocation. These are
```

Comment:
`by tel_id or EarthLocation` sounds like `EarthLocation` could be the key of the dictionary, not another format for the value.

Reply:
True, better with something like 

dict of telescope positions by tel_id, given as x,y,z position or `EarthLocation`.